init.zsh

```diff
@@ -9,20 +9,22 @@ if ! is-at-least 5.2; then
 fi
 
 # Define zim location
-ZIM="${ZDOTDIR:-${HOME}}/.zim"
+if [[ -z "${ZIM_HOME}" ]]; then
+  ZIM_HOME="${ZDOTDIR:-${HOME}}/.zim"
+fi
 
```

Comment:
Please test this with

    (( ! ${+ZIM_HOME} )) && ZIM_HOME="${ZDOTDIR:-${HOME}}/.zim"

See discussion in #180